src/Elliptic/Actions/InitializeAnalyticSolution.hpp

```diff
@@ -0,0 +1,74 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
```

Comment:
Please include 
`tuple`, `utility`, `TMPL.hpp`, `TaggedTuple`, `Tensor.hpp`, `ConstGlobalCache.hpp`

Reply:
should I include `TaggedTuple` even when the `inboxes` argument is unused? I forward-declare it currently